app/_how-tos/store-the-gateway-database-credentials-with-aws-secrets-manager.md

```diff
@@ -0,0 +1,144 @@
+  - on-prem
+
+min_version:
+  gateway: '3.4'
+
+tags:
+  - security
+  - secrets-management
+
+tldr:
+    q: How can I connect {{site.base_gateway}} to the database using credentials stored in AWS Secrets Manager?
+    a: |
+      Create a secret in AWS Secrets Manager with your PostgreSQL credentials, and start {{site.base_gateway}} with the required environment variables:
+        - `AWS_ACCESS_KEY_ID`, `AWS_SECRET_ACCESS_KEY`, `AWS_SESSION_TOKEN`, and `AWS_REGION` to connect to AWS
+        - `KONG_PG_USER` and `KONG_PG_PASSWORD`, where the values are references to your AWS secret
+      
+prereqs:
+  skip_product: true
+  inline:
+    - title: AWS configuration
+      content: |
+        This tutorial requires: 
+        - An AWS subscription with access to [AWS Secrets Manager](https://docs.aws.amazon.com/secretsmanager/latest/userguide/intro.html) and the following permissions:
+          - `secretsmanager:CreateSecret`
+          - `secretsmanager:PutSecretValue`
+          - `secretsmanager:GetSecretValue`
+        - [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/cli-chap-getting-started.html) installed
+        
+        You will also need the following authentication information to connect your AWS Secrets Manager with {{site.base_gateway}}:
+        - Your access key ID
+        - Your secret access key
+        - Your session token
+        - Your AWS region, `us-east-1` in this example
+
+        Create environment variables to store these credentials:
+        ```sh
+        export AWS_ACCESS_KEY_ID=your-aws-access-key-id
+        export AWS_SECRET_ACCESS_KEY=your-aws-secret-access-key
+        export AWS_SESSION_TOKEN=your-aws-session-token
+        export AWS_REGION="us-east-1"
+        ```
+      icon_url: /assets/icons/aws.svg
+
+cleanup:
+  inline:
+    - title: Clean up AWS resources
+      include_content: cleanup/third-party/aws
+      icon_url: /assets/icons/aws.svg
+    - title: Clean up Konnect environment
+      include_content: cleanup/platform/konnect
+      icon_url: /assets/icons/gateway.svg
+    - title: Destroy the {{site.base_gateway}} container
+      include_content: cleanup/products/gateway
+      icon_url: /assets/icons/gateway.svg 
+---
+
+## 1. Create a Docker network
+```sh
+docker network create kong-net
+```
+The Docker network will be used for communication between {{site.base_gateway}} and the database.
+
+## 2. Run the database
+Create the `kong-database` container for the PostgreSQL database: 
+```sh
+docker run -d --name kong-database \
+ --network=kong-net \
+ -p 5432:5432 \
+ -e "POSTGRES_USER=admin" \
+ -e "POSTGRES_PASSWORD=password" \
+ postgres:9.6
+```
+The username and password specified in this command are the PostgreSQL master credentials.
+
+## 3. Create environment variables
+Define the username and password to use to connect {{site.base_gateway}} to the database and store them in environment variables.
+```sh
+export KONG_PG_USER=kong
+export KONG_PG_PASSWORD=KongFTW
```

Comment:
```suggestion
export KONG_PG_PASSWORD=KongPassword
```
Just a suggestion, because I think FTW can be an abbreviation for something else